### PR TITLE
sendInvite

### PR DESCRIPTION
**Pull request by buaya123**

### CODE REVIEW

**Review by FotieMConstant — CHANGES_REQUESTED** [review]
Hey @buaya123, everything looks great. thanks for this amazing PR. please could you review the changes so we can merge?

Check the commented lines

File: okok/node-api/api.js [hunk 1]
```diff
@@ -62,179 +62,212 @@ const createUser = async (request, response) => {
...
+      }
+      // end of function that holds the formatted html invite
 
 
-  // encripting the sender's email to use in the invite
-  const cryptr = new Cryptr('bondkey');
-  const encryptedEmail = cryptr.encrypt(request.params.email);
+      // encripting the sender's email to use in the invite
+      const cryptr = new Cryptr('bondkey');
+      const encryptedEmail = cryptr.encrypt(receiver);
```
Comment (FotieMConstant) [inline okok/node-api/api.js:229]: Hey please, could you encrypt the sender's mail instead? Because we will need that later on remember?

Comment (FotieMConstant) [reply okok/node-api/api.js:229]: and also please could you rename `encryptedEmail` to `senderEncryptedEmail ` for better understanding?
Thanks

**Review by FotieMConstant — CHANGES_REQUESTED** [review]
Hey @buaya123, everything looks great. thanks for this amazing PR. please could you review the changes so we can merge?

Check the commented lines

File: okok/node-api/api.js (hunk 1 above)
Comment (buaya123) [reply okok/node-api/api.js:229]: cool2

**Comment by buaya123** [comment]
@FotieMConstant check

**Comment by buaya123** [comment]
hmm it still has an error

**Comment by FotieMConstant** [comment]
> hmm it still has an error

No it's all good on here

**Comment by FotieMConstant** [comment]
@buaya123 do you get an error or something? because all is good, I'm about to merge!

**Comment by buaya123** [comment]
Yeah im not getting my postman requests right

**Review by FotieMConstant — APPROVED** [review]

**Comment by buaya123** [comment]
although works on local but in preview it doesnt

**Comment by FotieMConstant** [comment]
> Yeah im not getting my postman requests right

@buaya123  What you getting? because it works just fine here!

**Comment by buaya123** [comment]
A server error has occurred

FUNCTION_INVOCATION_FAILED

**Pull request merged**